src/simplewallet/simplewallet.cpp

```diff
@@ -4468,6 +4485,23 @@ bool simple_wallet::show_transfer(const std::vector<std::string> &args)
       success_msg_writer() << "Timestamp: " << get_human_readable_timestamp(pd.m_timestamp);
       success_msg_writer() << "Amount: " << print_money(pd.m_amount);
       success_msg_writer() << "Payment ID: " << payment_id;
+      if (pd.m_unlock_time < CRYPTONOTE_MAX_BLOCK_NUMBER)
```

Comment:
I was referring to older wallet files that did not record the unlock time. The boost serialization code will default the `unlock_time` values on these older wallet files to zero on first load. So when loading an existing wallet, it will incorrectly show all prior transactions as having an unlock time on the genesis block. The log message below will then print an incorrect confirmations message. Treating zero as a special do-not-display-anything-case seems acceptable since no one will have an unlocked output on the genesis block.